conflowgen/__init__.py

```diff
@@ -63,6 +63,9 @@
 from conflowgen.analyses.container_flow_vehicle_type_adjustment_per_vehicle_analysis_report import \
     ContainerFlowVehicleTypeAdjustmentPerVehicleAnalysisReport
 
+# Cache for analyses and previews
+from conflowgen.data_summaries import DataSummariesCache
```

Comment:
When this class is officially part of the interface offered to users, it should also be documented - at least in the API part. However, I could not find it in https://conflowgen--183.org.readthedocs.build/en/183/api.html. Could you please add the documentation?

Reply:
See latest commit [Include DataSummariesCache in API documentation](https://github.com/1kastner/conflowgen/pull/183/commits/3534ada1091a6ff195668356de8306eb037b237e)